go/database/mpt/proof.go

```diff
@@ -201,6 +213,14 @@ func (p WitnessProof) String() string {
 	return b.String()
 }
 
+func (p WitnessProof) GetElements() []string {
+	res := make([]string, 0, len(p.proofDb))
+	for _, v := range p.proofDb {
```

Comment:
nit: for a future update, we might consider storing the elements inside the proof as strings instead of []byte.

Reply:
what is the benefit of using strings instead?

Reply:
You can skip the conversion step here and there, which is always a memory allocation and copy.